src/calculateRentalCost.js

```diff
@@ -29,6 +29,15 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const rental = days * 40;
+
+  if (days >= 3 && days < 7) {
+    return rental - 20;
+  } else if (days >= 7) {
+    return rental - 50;
+  } else {
+    return rental;
+  }
```

Comment:
You can remove last else:
```suggestion
  return rental;
```